api-report/driver-base.api.md

```diff
@@ -23,7 +23,7 @@ import { TypedEventEmitter } from '@fluidframework/common-utils';
 
 // @public
 export class DocumentDeltaConnection extends TypedEventEmitter<IDocumentDeltaConnectionEvents> implements IDocumentDeltaConnection, IDisposable {
-    protected constructor(socket: SocketIOClient.Socket, documentId: string, logger: ITelemetryLogger);
+    protected constructor(socket: SocketIOClient.Socket, documentId: string, logger: ITelemetryLogger, enableLongPollingDowngrades?: boolean);
```

Comment:
@vladsud you had mentioned each driver controlling policy. Is this what you had in mind? Or would it be better to add a protected property to DocumentDeltaConnection that could be set by the inheritor? Or some other way?

Reply:
Looks good! (as a first step, we might want to expose it in the future to driver factory users)